pkg/cloudprovider/gce/gce.go

```diff
@@ -154,6 +178,7 @@ func newGCECloud(config io.Reader) (*GCECloud, error) {
 		projectID:        projectID,
 		zone:             zone,
 		instanceID:       instanceID,
+		networkName:      networkName,
```

Comment:
Agree with @bgrant0607, but I think this is fine for this PR. I'll plumb a node network name through the master in a separate PR.